client/src/utils/parser.test.js

```diff
@@ -0,0 +1,351 @@
+const parser = require('./parser');
+const JSON_FOR_TESTING = {
```

Comment:
@SanJSp  @WolfgangDaniel Please let me know ho you think about moving ths example json to a separate (helper) file and load it from there. I think that could be a good thing to do. Please let me know @WolfgangDaniel where that file could be located accourding to our structure, as I am a bit unsure about that

Reply:
In my opinion really good thing to do, always outsource constants. Because this is a big constant, a new file is a great idea